internal/agent/install.go

```diff
@@ -236,6 +240,53 @@ func RunInstall(c *config.Config) error {
 
 // runInstallUki runs the UKI path install
 func runInstallUki(c *config.Config) error {
+	// Check if we are running in PXE
+	efivar, err := internalutils.ReadEfivar(constants.PXEVarFile)
+	// We dont care if we fail to read it, that means its not there
+	if err == nil {
```

Comment:
This belongs to a separate function I think

Reply:
done!